src/components/Card.js

```diff
@@ -1,6 +1,7 @@
 import styled from "styled-components/macro";
 import { useState } from "react";
 import JoinButton from "./JoinButton";
+import { VscTrash } from "react-icons/vsc";
```

Comment:
You could rename this icon for better understanding. Maybe `import {VscTrash as delete} from "react-icons/vsc"`?

Reply:
Thx, that's a really good idea! I changed it to {VscTrash as DeleteIcon}. 🤩